docs/en/reference/association-mapping.rst

```diff
@@ -16,6 +16,8 @@ This chapter is split into three different sections.
 - :ref:`association_mapping_defaults` are explained that simplify the use-case examples.
 - :ref:`collections` are introduced that contain entities in associations.
 
+One tip for working with relations is to read the relation from left to right, where the left word referece to the current class.
```

Comment:
Can you add some wording examples here? Such as "OneToMany" means "One" (this side), to "Many" (referenced side).